ui/src/app/machines/views/MachineList/MachineListTable/MachineListTable.js

```diff
@@ -406,337 +406,334 @@ const generateGroupRows = ({
   return rows;
 };
 
-const MachineListTable = ({
-  filter,
-  grouping,
-  hiddenGroups,
-  setHiddenGroups,
-}) => {
-  const dispatch = useDispatch();
-  const selectedMachines = useSelector(machineSelectors.selected);
-  const selectedIDs = useSelector(machineSelectors.selectedIDs);
-  const machines = useSelector((state) =>
-    machineSelectors.search(state, filter, selectedIDs)
-  );
+const MachineListTable = React.memo(
```

Comment:
The way I've seen this done is to instead change the export below, something like:
```js
export default React.memo(MachineListTable);
```

The docs show both ways: https://reactjs.org/docs/react-api.html#reactmemo

I think one of the benefits of applying the memo to the default export is that we can still export the un-memoized component for testing if need be.